cfme/common/provider.py

```diff
@@ -74,9 +74,14 @@ def form(self):
                 "Default": [
                     ('default_principal', Input("default_userid")),
                     ('default_secret', Input("default_password")),
-                    ('default_verify_secret', Input("default_verify")),
```

Comment:
I'm afraid you can't remove `default_verify_secret` - this might not cause failure with container providers but it will make all the other fail.